ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-icmp.j2

```diff
@@ -0,0 +1,21 @@
+{% if ip_icmp is defined and ip_icmp is not none %}
+
+## IP ICMP
+
+### IP ICMP Settings
+
+IP ICMP Redirect: {{ ip_icmp.redirect }}
+IP ICMP Rate Limit Unreachable: {% if ip_icmp.rate_limit_unreachable %}0{% else %}Disabled{% endif%}
```

Comment:
missing test for variable. I suggest using `if ip_icmp.rate_limit_unreachable is arista.avd.defined(true)`
Please ensure consistent spacing `{%<space>` and `<space>%}`